test/test_utils.py

```diff
@@ -42,3 +43,54 @@ def test_json_encoder():
     data2 = {'np_list': np.array(['a', 'b', 'c'])}
     data2_e = {'np_list': ['a', 'b', 'c']}
     eq_(json.dumps(data2_e, indent=0), json_dumps(data2))
+
+
+class MockChart(object):
+    """
+    A mock class for a Chart and Page
+    """
+
+    def __init__(self, js_dependencies):
+        self.js_dependencies = js_dependencies
+
+
+def test_merge_js_dependencies():
```

Comment:
Again, sorry that I may have started the lengthy test case previously. but it is better to split the test case into different test cases. Too many assertion in one test case is not recommended.